resolwe_runtime_utils.py

```diff
@@ -359,6 +575,8 @@ def import_file(
     :return: Destination file path (if extracted and compressed, extracted path given)
     """
 
+    import requests
```

Comment:
I don't like the idea of using `requests` in arbitrary docker images. But this is not the problem with this pull request; just keep in mind that we should fix this in the future.

Reply:
It is only used to download external files. Maybe the import should be moved to the section that is executed only if external file download is really requested?